demo/docker-compose.yaml

```
@@ -1,14 +1,14 @@
 version: '3.9'
 
 services:
-  helix:
-    container_name: helix
-    image: registry.helix-dev.synadia.io/helix:tag-release
-    hostname: helix
-    command: ["-c", "/conf/helix/rna.cue", "server", "start"]
+  control-plane:
+    container_name: control-plane
+    image: registry.helix-dev.synadia.io/control-plane:tag-rc
```

Prep for the release, will merge after tagged

```suggestion
    image: registry.helix-dev.synadia.io/control-plane:tag-release
```